CHANGELOG.rst

```diff
@@ -178,6 +178,7 @@ Bug Fixes
 - Add support for rotation parameters to `sunpy.map.make_fitswcs_header`. (`#3139 <https://github.com/sunpy/sunpy/pull/3139>`__)
 - Improve the implementation of `~sunpy.physics.differential_rotation.differential_rotate` the image warping when transforming Maps for differential rotation and change in observer position. (`#3149 <https://github.com/sunpy/sunpy/pull/3149>`__)
 - Fix a bug where new helioviewer sources potentially cause `~sunpy.net.helioviewer.HelioviewerClient.data_sources` to error. (`#3162 <https://github.com/sunpy/sunpy/pull/3162>`__)
+- Updated the `sunpy.map.compositemap` to support linewidths argument. (`#3792 <https://github.com/sunpy/sunpy/pull/3792>`__)
```

Comment:
We don't use this file directly. You have to add a changelog file in the `changelog/` folder.

Reply:
I have done the required changes.

Reply:
Thanks, can you merge in the current version of sunpy master? The tests are all failing due to the old base this PR is using.

Reply:
I have merged it into the master branch.